sdk/attestation/azure-security-attestation/tests/test_policy_getset_async.py

```diff
@@ -17,273 +17,387 @@
 # ----------------------
 
 from logging import fatal
-from typing import ByteString
+import asyncio
+from typing import Any, ByteString
 import unittest
 from cryptography.hazmat.primitives import hashes
 from devtools_testutils import AzureTestCase, PowerShellPreparer
```

Comment:
For the `typing` import, same comment as in the sync tests
```suggestion
from typing import Any
from cryptography.hazmat.primitives import hashes
from devtools_testutils import AzureTestCase
```